### PR TITLE
Fixes #1733 Comparison of Expression Profiles

### PR DESCRIPTION
**Pull request by rwmcintosh**

### CODE REVIEW

File: src/MoBi.Core/CoreRegister.cs [hunk 1]
```diff
@@ -92,11 +93,14 @@ public override void RegisterInContainer(IContainer container)
          registerCommitTasks(container);
 
          registerConverters(container);
+
+         // Temporary registration while Core components reside in MoBi
+         container.AddRegister(x => x.FromType<MoBiRegisterOfCore>());
```
Comment (msevestre) [inline src/MoBi.Core/CoreRegister.cs:98]: Good idea

**Review by msevestre — APPROVED** [review]
Works. Wondering about the implementation since most of it will be deleted? What's left at the end?

**Comment by rwmcintosh** [comment]
> Works. Wondering about the implementation since most of it will be deleted? What's left at the end?

There will just be these 3 DiffBuilders all based on what is now called StartValuesDiffBuilder or something.

**Comment by rwmcintosh** [comment]
> Works. Wondering about the implementation since most of it will be deleted? What's left at the end?

If you mean, why did you paste that code - it's because the StartValuesDiffBuilder is an internal class in Core, not protected or public

**Pull request merged**